### PR TITLE
Adds more quotes to the EXTREMELY POPULAR skeleton toys

### PR DESCRIPTION
**Pull request by coiax**

### CODE REVIEW

**Comment by AdamElTablawy** [comment]
>skeletor 

just more puns

**Comment by Dorsisdwarf** [comment]
Does not actually play MYAAAAAAAAAAH /10

**Comment by Dorsisdwarf** [comment]
I just remembered that we have a rule about PR titles being accurate or something

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -618,7 +618,22 @@
 		"I'm a fighter, not a liver.",
 		"How can I see without eyeballs?",
 		"Ask your parents about 'boning', before you get pregnant.",
-		"Remember, a dog is for life, not just for christmas.")
+		"Remember, a dog is for life, not just for christmas.",
+
```
Comment (RemieRichards) [inline code/game/objects/items/toys.dm]: can you not mr/mrs spooky line that is empty.

**Comment by lzimann** [comment]
but there's nothing skeleton-related

**Comment by PKPenguin321** [comment]
"I'd ask if you want a bone, but ERP is banned."

**Comment by Cheridan** [comment]
are these more understale references, i don't know because i don't play meme games.

please respond

**Comment by KorPhaeron** [comment]
@Cheridan 

I think it's a mix of undertale and skeletor quotes

**Comment by Cheridan** [comment]
oh ok cool

**Pull request closed without merging**